app/SearchAPI/SearchAPIServiceProvider.php

```diff
@@ -20,6 +20,7 @@ public function register(Container $pimple)
                     'base_uri' => $pimple['search_api.base_url'],
                     'headers' => [
                         'X-Api-Key' => $pimple['search_api.api_key'],
+                        'X-Client-Properties' => 'cluster|widgets, cluster|snowplow',
```

Comment:
It probably doesn't matter but in the ticket it is written in small letters.
`x-client-properties`

Reply:
An HTTP header consists of its case-insensitive name

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

I matched it with the one above to be consistent 😃

Reply:
😂